### ISSUE TITLE
Enhancement to be able to simultanenously pull models from two locations at the same time 

### ISSUE THREAD

**Issue by RitchZoeSquad** [n4ze3m/page-assist#267]
I have two pcs running ollama and I'd like to be able to interact with both pcs at the same time

Either we would need a location to add both host so that it can pull one entirely or have a section to choose which host to interact with.

**Comment by n4ze3m** [comment]
Hey, currently you can add one main Ollama, and the rest from OpenAI-compatible endpoints section 🤔

**Comment by RitchZoeSquad** [comment]
But would that allows me to see 10 models from host one and 11 from host 2?

**Comment by n4ze3m** [comment]
Yes, as far as I know. Let me know if you encounter any errors; I will improve it. Currently, multiple Ollama instances are an experimental feature. :)
